packages/server-fetch/src/parsers.ts

```diff
@@ -32,18 +32,18 @@ const jsonParser = (options: ExtendedFetchOptions) => {
 	return options as FetchOptions;
 };
 
-const urlencodedParser = (options: ExtendedFetchOptions) => {
+// Do not parse anything. Just return the options as they are.
+// This will be used for binary data, for example.
+const defaultParser = (options: ExtendedFetchOptions) => {
 	return options as FetchOptions;
 };
 
 const getParser = (contentTypeHeader?: string): ((options: ExtendedFetchOptions) => FetchOptions) => {
 	switch (contentTypeHeader) {
 		case 'application/json':
 			return jsonParser;
-		case 'application/x-www-form-urlencoded':
-			return urlencodedParser;
 		default:
-			return jsonParser;
```

Comment:
I'm trying to think on how changing the default parser could affect other calls. 

For now, maybe adding a new case for the type of data you're sending works? 👀